tests/app/service/test_rest.py

```diff
@@ -1460,7 +1452,7 @@ def test_get_detailed_services_for_date_range(
         utc_date=(datetime.utcnow() - timedelta(days=1)).date(), service=template.service, notification_type='sms'
     )
 
-    notification = create_notification(template=template, created_at=datetime.utcnow(), status='delivered')
+    notification = sample_notification(template=template, created_at=datetime.utcnow(), status='delivered')
```

Comment:
This doesn't need to be assigned because using the fixture means the associated teardown is no longer necessary.
```suggestion
    sample_notification(template=template, created_at=datetime.utcnow(), status='delivered')
```

Reply:
Updated.